system/core.go

```diff
@@ -97,10 +101,13 @@ func (application *Application) LoadTemplates(templatePath string) error {
 	return err
 }
 
+// Close prints "Application.Close() called"
```

Comment:
Wtf is this func... this is worth investigating, because this is very strange. Don't do it in this PR though, this one is already uncomfortably big.

Reply:
My first thought was that this is needed to satisfy some interface, but I'm not sure what.